fla/ops/linear_attn/utils.py

```
@@ -4,7 +4,9 @@
 
 
 @torch.jit.script
-def normalize_output(q, k, o):
+def normalize_output(q, k, o, cum_k=None):
     k = k.cumsum(-2)
+    if cum_k is not None:
+        k=k+cum_K
```

Ruff (0.8.2)

Undefined name `cum_K`

(F821)